python/ray/worker.py

```diff
@@ -636,6 +637,9 @@ def init(
         _metrics_export_port: Optional[int] = None,
         _system_config: Optional[Dict[str, str]] = None,
         _tracing_startup_hook: Optional[Callable] = None,
+        _raylet_socket_name:_raylet_socket_name[str] = None,
+        _plasma_store_socket_name: Optional[str] = None,
+        _redis_max_clients: Optional[int] = None,
```

Comment:
From our comment left in 0.8.4: "ray defaults redis_max_clients to `ulimit -n` - 32, this value can be extremely large in docker, so we manually cap it to the same value than our test EC2 instances"
I haven't found signs that this has changed, so keeping this parameter out of safety.